label_maker/package.py

```diff
@@ -73,25 +74,37 @@ def package_directory(dest_folder, classes, imagery, ml_type, seed=False,
 
     # open the images and load those plus the labels into the final arrays
     if is_tif(imagery):  # if a TIF is provided, use jpg as tile format
-        image_format = '.jpg'
+        img_dtype = rasterio.open(imagery).profile['dtype']
+        try:
+            # MS bands written to png
+            kwargs['band_indicies']
+            image_format = '.png'
+        except KeyError:
+            # 3 band RGB written to jpg
+            image_format = '.jpg'
```

Comment:
Why do we need to write to different image formats if band indices is set? Also what happens if we select five indices?